in_toto/runlib.py

```diff
@@ -823,6 +860,7 @@ def in_toto_record_stop(step_name, product_list, signing_key=None,
   else:
     unfinished_fn_glob = UNFINISHED_FILENAME_FORMAT_GLOB.format(
         step_name=step_name, pattern="*")
+
```

Comment:
Stray line that probably snuck in during one of the prior discussions about where to put the unfinished link.